sdk/core/azure_core_test/src/recording.rs

```diff
@@ -220,7 +220,7 @@ impl Recording {
     /// ```
     /// # let recording = azure_core_test::Recording::with_seed();
     /// let dek: [u8; 32] = recording.random();
-    /// # assert_eq!(typespec_client_core::base64::encode(dek), "HumPRAN6RqKWf0YhFV2CAFWu/8L/pwh0LRzeam5VlGo=");
+    /// # assert_eq!(azure_core::base64::encode(dek), "HumPRAN6RqKWf0YhFV2CAFWu/8L/pwh0LRzeam5VlGo=");
```

Comment:
Don't know why - or maybe it was - this wasn't breaking other PRs. @jhendrixMSFT I believe it was cleaned up a bunch of the `typespec_client_core` refs we didn't want in anything but `azure_core`, which I agreed with and approved. But this should've broken then. Odd. Otherwise unrelated to the PR.